src/PrestaShopBundle/Exception/UpdateProductException.php

```diff
@@ -33,4 +33,5 @@
  */
 class UpdateProductException extends Exception
 {
+    public const FAILED_BULK_UPDATE_STATUS = 10;
```

Comment:
That's an old exception used in legacy page You shouldn't rely on this one but rather on the new ones from the product domain:
https://github.com/PrestaShop/PrestaShop/tree/3d06748975e00c548e940bf82862d9d211ae94d4/src/Core/Domain/Product/Exception